src/accounts/password.js

```diff
@@ -0,0 +1,62 @@
+import * as bcrypt from 'bcrypt';
+import getAccountDb, { clearExpiredSessions } from '../account-db.js';
+
+function hashPassword(password) {
+  return bcrypt.hashSync(password, 12);
+}
+
+export function bootstrapPassword(password) {
+  if (!password || password === null || password === '') {
+    return { error: 'invalid-password' };
+  }
+
+  getAccountDb().mutate('DELETE FROM auth WHERE method = ?', ['password']);
+
+  // Hash the password. There's really not a strong need for this
+  // since this is a self-hosted instance owned by the user.
+  // However, just in case we do it.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏hashing passwords is security 101. Even for local set-ups that should be done. Lets remove the comment.
```suggestion
```

Reply:
agreed. this probably comes from the old PR